### DISCUSSION TITLE
auto pull request.

### DISCUSSION THREAD

**Discussion by 12saif** [community/community#48506] [Pull Requests] [answered]
### Select Topic Area

General

### Body

**how to set auto pull request and merge request with setting.**

**Comment by CodeMaster7000** [comment, marked as answer]
@12saif 

1. In the repository's main page, go to "Pull Requests".
2. Click on the PR you would like to auto-merge.
3. Click on the "Enable auto-merge" drop-down menu, then click a merge method.
4. Click "Enable auto-merge".

Voila!

**Reply by 12saif** [reply]
Thanks for answer
